src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  max-width: calcCalendarWidth($day-width-height, $gap);
+  display: flex;
+  flex-wrap: wrap;
+
+  &__day {
+    @include square($day-width-height);
+    background-color: $color-day-bg;
+    border: 1px solid $color-day-border;
+    display: flex;
+    margin-right: $gap;
+    margin-bottom: $gap;
+    cursor: pointer;
+    font-family: "Arial", sans-serif;
+    font-size: 30px;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: #fcbfcb;
+      transform: translateY(-20px);
+    }
+
+    @for $day-number from 1 through 31 {
+      &:nth-child(7n) {
+        margin-right: 0;
+      }
+
+      &:nth-child(#{$day-number}):after {
+        content: "#{$day-number}";
+        margin: auto;
+      }
+    }
+  }
+
+  @each $day-name, $day-num in $starting-day-names {
+    &--starting-day-#{$day-name} &__day:first-child {
+      margin-left: calcFirstDayMarginLeft($day-width-height, $day-num, $gap);
+    }
+
+    &--starting-day-#{$day-name} &__day:nth-child(7n) {
+      margin-right: $gap;
+    }
+
+    &--starting-day-#{$day-name} &__day:nth-child(7n + calc(7 - #{$day-num})) {
+      margin-right: 0;
+    }
```

Comment:
i see that you made setting gaps by margins in calendar difficult and hard to understand. maybe it would be simpler to use `gap: 1px` in `.calendar` since we don't have any tests. how do you think?

Reply:
👍You're right, it's an easier solution

Reply:
> i see that you made setting gaps by margins in calendar difficult and hard to understand. maybe it would be simpler to use `gap: 1px` in `.calendar` since we don't have any tests. how do you think?

👍You're right, it's an easier solution